### ISSUE TITLE
Linking monitors to monitor groups

### ISSUE THREAD

**Issue by lauraseidler** [wasfree/terraform-provider-uptrends#57]
Hi,

we just noticed that while we can create a `uptrends_monitor_group` resource, there doesn't seem to be a way to actually link monitors to these groups? Is this something that's planned? Are you accepting PRs?

Thanks!

**Comment by jayenzo** [comment]
Yes I was wondering the same thing I don't see any section were we can add monitors to a specific group
